greyhounds/zoya.md

```diff
@@ -5,4 +5,14 @@ greyhound: zoya
 permalink: /greyhounds/zoya/
 ---
 
-I've just arrived!
+Hi! My name is Zoya! 
+
+I've recently retired from the fast-paced world of being a professional greyhound, and I love lounging, snuggling, and leaning against the nearest human. I'm not especially food-motivated, but I sure wasn't kidding about snuggling-- nothing makes me feel safer and happier than curling up next to you for some scritches and pets, or lying down for a nice long brushing.
+
+I get along well with other (medium-big) dogs. In fact, I'm a little nervous when I'm the only dog around. You won't have to worry about doing basic training with me as I am already house-trained and crate trained. I'm a quick learner about house rules, but I'd also love to learn some commands-- I'm not sure what "sit" or "stay" mean yet, but I'm excited to find out!
+
+I get a little nervous sometimes when there's a lot of sensory stimulation because I'm still adjusting to life with humans, but mostly I'm just a sweetheart looking for a loving home.
+
+My perfect forever home would be a house full of people to pet and snuggle me, but that mostly want to watch me sleep. I'd love to join your pre-existing pack of greyhounds or other medium-large size calm dogs. 
+
+My foster parents are pretty obsessed with taking photos of me, so I am thinking of pursuing a modeling career in my retirement. Check out my headshots! 
```

Comment:
So the bio files are written using Markdown syntax. [Here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) is a cheat sheet for Markdown.

Links are written as so: \[Link name\]\(http:link.url\)

You can add another commit to this pull request to add the link.